src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -239,18 +271,20 @@ void func_80A74398(Actor* thisx, PlayState* play) {
     }
 }
 
-s32 func_80A745E4(EnIk* this, PlayState* play) {
-    if (((this->unk_2FB != 0) || (this->actor.params == 0)) &&
+// Checks to see if action can be interrupted to block attack
+s32 EnIk_ChangeAction(EnIk* this, PlayState* play) {
+    if (((this->armorStatusFlag != 0) || (this->actor.params == IK_TYPE_NABOORU)) &&
         (func_800354B4(play, &this->actor, 100.0f, 0x2710, 0x4000, this->actor.shape.rot.y) != 0) &&
         (play->gameplayFrames & 1)) {
-        func_80A755F0(this);
+        EnIk_SetupStopAndBlock(this);
         return true;
     } else {
         return false;
     }
 }
 
-Actor* func_80A74674(PlayState* play, Actor* actor) {
+// Checks to see if the Actor is either the pillars or chairs (IronObj)
+Actor* EnIk_CheckForBgJyaIronObj(PlayState* play, Actor* actor) {
```

Comment:
```suggestion
Actor* EnIk_FindBgJyaIronObj(PlayState* play, Actor* actor) {
```
I think find is the more common terminology for these types of functions.